README.md

```diff
@@ -53,6 +53,10 @@ selecting the `Use DiCy` option in the settings page. [DiCy][] will be installed
 automatically and so no further action is required for either TeX Live or
 MiKTeX.
 
+### Code Highlighting
+In order to properly work in Atom with this package, you'll need a code
+highlighting extension, namely [`language-latex`][language-latex-pkg].
+
```

Comment:
I propose that this paragraph be changed to the following:
``` md
In order for this package to behave as expected, your Atom environment must
contain a package that provides a LaTeX grammar. We suggest
[`language-latex`][language-latex-pkg], but other valid options might exist.
``` 
What do you think? /cc @yitzchak